lib/syskit/network_generation/logger.rb

```diff
@@ -82,6 +82,21 @@ def configure
                 end
             end
 
+            @@logfile_indexes = {}
+
+            def self.default_log_file_name(name)
+                name[/.*(?=_[L|l]ogger)/] || name
+            end
+
+            def self.setup_default_logger(logger, log_file_name: default_log_file_name(logger.orocos_name))
+                index = (@@logfile_indexes[log_file_name] ||= -1) + 1
+                @@logfile_indexes[log_file_name] = index
+                log_file_path = "#{log_file_name}.#{index}.log"
+
+                logger.properties.file = log_file_path
+                logger
```

Comment:
Just `logger.properties.file = "#{log_file_name}.#{index}.log"` would be sufficient

- the method does not have to return `logger` (if it should return something, it would be the log file name IMO)
- log_file_path is not used anywhere else, there's no real point creating a local variable.